src/components/home/StoryUpload.jsx

```diff
@@ -234,7 +234,7 @@ const StoryUpload = ({ onSuccess = () => {} }) => {
                           const selectedOption = storyWorldOptions?.find(option => option._id === e.target.value);
                           setFieldValue('storyWorld', e.target.value);
                           setFieldValue('storyWorldLead', selectedOption ? selectedOption.lead_who : '');
-                          setFieldValue('storyLeadWho', selectedOption ? selectedOption.lead_who : '');
```

Comment:
why is this removed

Reply:
As You said for that we need blank value

Reply:
ok got it